src/Microsoft.AspNet.Mvc.Core/ModelBinding/Validation/ValidationVisitor.cs

```diff
@@ -191,11 +182,16 @@ private bool VisitEnumerableType()
         {
             var isValid = true;
 
-            if (_model != null)
+            if (_model != null && _metadata.ValidateChildren)
             {
                 var strategy = _strategy ?? DefaultCollectionValidationStrategy.Instance;
                 isValid = VisitChildren(strategy);
             }
+            else if (_model != null)
+            {
+                // Suppress validation for the prefix, but we still want to validate the object.
```

Comment:
Think the comment would be more helpful if it were `Suppress validation for the entries. May set the model's entry to Skipped but subsequent code will validate that collection.`